src/shell/shell_misc.cpp

```diff
@@ -97,7 +107,7 @@ void DOS_Shell::InputCommand(char * line) {
 
 				case 0x4B: /* Left */
 					if (str_index) {
-						outc(8);
+						backone();
```

Comment:
`back` can be a noun, and without context, it's not obvious what 'backone' is operating on.
Suggest clarifying with a verb and use underscore separation: (maybe?)  `move_cursor_back_one()`, or something like that 👍

Reply:
The function name "move_cursor_back_one" is more readable. Done.